src/po.h

```diff
@@ -1,6 +1,9 @@
 #ifdef ENABLE_NLS
 #include <libintl.h>
 #define _(String) dgettext("data.table", String)
+// NB: flip argument order to match that of R's ngettext()
+#define Pl_(n, String1, StringPlural) dngettext("data.table", String1, StringPlural, n)
```

Comment:
`dngettext` accepts an `unsigned long`. I think that all the lengths given to it so far are `int`s and have good reason to stay nonnegative `int`s. So it's worth mentioning `((n) > ULONG_MAX ? ((n) % 1000000) + 1000000 : (n)))` from the [gettext documentation](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html), but we probably don't need it.

Reply:
Let's leave it be for now, great flag & find though.